components/list/list.js

```diff
@@ -0,0 +1,71 @@
+import './list-item.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+
+class List extends LitElement {
+	static get properties() {
+		return {
+			divider: { type: String }
+		};
+	}
+
+	static get styles() {
+		const layout = css`
+			:host {
+				display: block;
+			}
+			ul {
+				list-style-type: none;
+				padding: 0;
+			}
+			::slotted(*) {
+				display: none;
+			}
+			::slotted(d2l-list-item) {
+				--d2l-list-item-divider-bottom: block;
+				--d2l-list-item-divider-top: block;
+				box-sizing: content-box;
+				display: list-item;
+				margin: 1px 0;
```

Comment:
Why is the outer list applying `box-sizing`, `display` and `margin` to the list items? Shouldn't that be their job?

Reply:
I could go either way on this and did flip back and forth.

The reason I ended up doing it this way was because I felt these styles described how it fit within the list container.